### ISSUE TITLE
LoG kernel equation is not correct

### ISSUE THREAD

**Issue by chinnl** [lwCVer/LEGNet#6]
Hi, I was attaching your stem block to top of another model and got NaN losses, then I'd found that your LoG kernel equation may be not correct. Your equation: 
`kernel = (xx**2 + yy**2 - 2 * sigma**2) / (2 * math.pi * sigma**4) * torch.exp(-(xx**2 + yy**2) / (2 * sigma**2))`
should be
`kernel = (xx**2 + yy**2 - 2 * sigma**2) / (2 * math.pi * sigma**6) * torch.exp(-(xx**2 + yy**2) / (2 * sigma**2))`.
Is this a mistake or you changed the exponent of sigma for some reasons?
Btw, this kernel with kernel_size = 7, sigma = 1 (as your default setting) has mean and sum very close to 0. So the normalization:
`kernel = kernel - kernel.mean()
kernel = kernel / kernel.sum()`
make the elements' values become very large, cause exploring gradient and NaN box/class loss in training.

**Comment by lwCVer** [comment]
Thank you for your interest in our work!
Regarding the equation error: You are correct that sigma**4 should be sigma**6. This was likely a typo. In our paper and code, we set sigma=1, which minimized the impact of this error.
Regarding the normalization issue: For a "derivative-like" kernel such as LoG, normalizing by dividing by its sum is inappropriate and can cause numerical instability. This explains the exploding gradients and NaN loss issues you encountered.
Thank you for carefully pointing out these potential issues. Your question is very professional, and your analysis is correct. We look forward to the improvements in your future work.
